src/components/widgets/WaterTemperatureWidget.tsx

```diff
@@ -0,0 +1,38 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback } from 'react';
+
+import { Icon, normalize, texts } from '../../config';
+import { useHomeRefresh } from '../../hooks';
+import { useWaterTemperature } from '../../hooks/waterTemperature';
+import { QUERY_TYPES } from '../../queries';
+import { ScreenName, WidgetProps } from '../../types';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const WaterTemperatureWidget = ({ text, additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+  const { temperature, refresh } = useWaterTemperature();
+
+  const onPress = useCallback(() => {
+    navigation.navigate(ScreenName.Html, {
+      title: text ?? texts.waterTemperature.headerTitle,
+      query: QUERY_TYPES.WATER_TEMPERATURE,
+      queryVariables: { name: additionalProps?.staticContentName ?? 'water-temperature' }
```

Comment:
i am thinking about removing the fallback. if someone forgets to addd a static content name, then there is something missing. we cannot achieve a fallback static content with `water-temperature` for every city.

Reply:
I can add a control for that. If `staticContentName` is not added, no redirection is done.

Reply:
we do not need that. if user setup a widget, they need to setup all necessary parts. this is not created automatically or with something dynamical.